docs/spec/v1beta2/kustomization.md

```diff
@@ -1452,9 +1300,9 @@ them on the cluster.
 For secrets in `.json`, `.yaml` and `.env` format, make sure you specify the input type when encrypting them with SOPS:
```

Comment:
This should probably also mention INI for sake of completeness, as this is the forth supported [store format](https://github.com/mozilla/sops/tree/master/stores).

Reply:
There is no INI generator in Kustomize, only `env`. So for `ini` people should use the binary format.